reactfire/firestore/index.tsx

```diff
@@ -44,7 +44,7 @@ export function useFirestoreDoc<T = unknown>(
 ): T extends {} ? T : firestore.DocumentSnapshot {
   return useObservable(
     doc(ref),
-    'firestore doc: ' + ref.path,
+    `useFirestoreDoc:${ref.path}:${JSON.stringify(options)}`,
```

Comment:
this should have the same observableId as `preloadFirestoreDoc` above so that it can benefit from the eager fetch